roles/openshift_common/README.md

```diff
@@ -20,6 +20,7 @@ Role Variables
 | openshift_ip              | UNDEF             | Internal IP address to use for this host    |
 | openshift_public_hostname | UNDEF             | Public hostname to use for this host        |
 | openshift_public_ip       | UNDEF             | Public IP address to use for this host      |
+| openshift_portal_net      | 172.30.0.0/16     | Service IP CIDR |
```

Comment:
Do these README generally refer to the actual variable or the fact that they alter?